From 3c4d1203eb9ee3dbe79b096d587c9baaf8b802ed Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Fri, 11 Mar 2022 13:26:31 -0700
Subject: [PATCH] Fix flit_core build requires/backend.

Only flit_core should be required by pyproject.toml, the regular flit
package is the pep517 frontend which is not what should be set for the
build system.

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
Upstream: https://github.com/aio-libs/aiologstash/pull/258
---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 7bdaebb..252a660 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["flit"]
-build-backend = "flit.buildapi"
+requires = ["flit_core"]
+build-backend = "flit_core.buildapi"
 
 [tool.flit.metadata]
 module = "aiologstash"
-- 
2.34.1

